packages/components/src/CollapsiblePanel/CollapsiblePanel.component.js

```diff
@@ -148,7 +148,7 @@ function renderHeader(header, content, onSelect, onToggle) {
 	if (content) {
 		const defaultCaret = (
 			<Button
-				className={css.toggle}
+				className={classNames(css.toggle, 'toggle-button')}
```

Comment:
According to PSD, arrow must be in grey
![adobe creative cloud](https://cloud.githubusercontent.com/assets/18534166/25011501/dba32bde-206d-11e7-8a60-1d5d2cbc946a.png)